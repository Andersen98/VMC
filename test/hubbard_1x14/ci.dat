cicpsslater
correlator 3 threesite.txt
maxIter 10
stochasticIter 5000
seed 5

#stepsize 0.01
#decay2 0.01
#restart

