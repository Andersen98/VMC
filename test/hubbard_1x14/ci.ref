cicpsslater
correlator 3 threesite.txt
maxIter 10
stochasticIter 5000
seed 5

#stepsize 0.01
#decay2 0.01
#restart


    0   -10.77921461 (2.59e-02)     0.76822329      8.9       5000     4.23
    1   -10.84412235 (2.88e-02)     0.83134031     10.2       5000     8.53
    2   -10.85133680 (2.71e-02)     0.97159156     10.7       5000    12.82
    3   -10.86921470 (2.45e-02)     1.03854463     10.0       5000    17.11
    4   -10.85225204 (2.29e-02)     1.89900506      9.6       5000    21.39
    5   -10.95128849 (2.21e-02)     1.36438539      8.8       5000    25.70
    6   -10.99078262 (1.97e-02)     1.28111646      7.5       5000    30.02
    7   -10.97499071 (2.09e-02)     2.25794428     10.3       5000    34.31
    8   -11.01044956 (2.03e-02)     0.94697587      9.5       5000    38.62
    9   -10.96667482 (2.18e-02)     0.97134898     11.2       5000    42.99
