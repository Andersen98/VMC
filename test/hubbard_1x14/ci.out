cicpsslater
correlator 3 threesite.txt
maxIter 10
stochasticIter 5000
seed 5

#stepsize 0.01
#decay2 0.01
#restart


    0   -10.77921461 (2.59e-02)     0.76822329      8.9       5000     4.22
    1   -10.84412235 (2.88e-02)     0.83134031     10.2       5000     8.54
    2   -10.85133680 (2.71e-02)     0.97159156     10.7       5000    12.84
    3   -10.86921470 (2.45e-02)     1.03854463     10.0       5000    17.13
    4   -10.85225204 (2.29e-02)     1.89900506      9.6       5000    21.41
    5   -10.95128849 (2.21e-02)     1.36438539      8.8       5000    25.73
    6   -10.99078262 (1.97e-02)     1.28111646      7.5       5000    30.03
    7   -10.97499071 (2.09e-02)     2.25794428     10.3       5000    34.32
    8   -11.01044956 (2.03e-02)     0.94697587      9.5       5000    38.64
    9   -10.96667482 (2.18e-02)     0.97134898     11.2       5000    42.94
